packages/faustwp-cli/index.ts

```diff
@@ -3,13 +3,28 @@
 import { spawn } from 'child_process';
 import dotenv from 'dotenv-flow';
 
-import { getCliArgs, validateFaustEnvVars } from './utils/index.js';
+import {
+  getCliArgs,
+  validateFaustEnvVars,
+  generatePossibleTypes,
+} from './utils/index.js';
 
-dotenv.config();
-validateFaustEnvVars();
+// eslint-disable-next-line func-names
+await (async function () {
+  dotenv.config();
+  validateFaustEnvVars();
 
-/**
- * Spawn a child process using the args captured in argv and continue the
- * standard i/o for the Next.js CLI.
- */
-spawn('next', getCliArgs(), { stdio: 'inherit' });
+  const arg1 = getCliArgs()[0];
+
+  // Handle custom CLI arguments.
+  if (arg1 === 'generatePossibleTypes') {
+    await generatePossibleTypes();
+    process.exit(0);
+  }
```

Comment:
Removed the switch case I had earlier in this PR as too many adjustments were needed to satisfy the linter